src/Security/BasicAuth.php

```diff
@@ -82,10 +85,12 @@ public static function requireLogin($realm, $permissionCode = null, $tryUsingSes
 
         $member = null;
         if (isset($_SERVER['PHP_AUTH_USER']) && isset($_SERVER['PHP_AUTH_PW'])) {
-            $member = MemberAuthenticator::authenticate(array(
+            $authenticator = Injector::inst()->get(Authenticator::class);
+
+            $member = $authenticator->authenticate([
                 'Email' => $_SERVER['PHP_AUTH_USER'],
```

Comment:
Are we planning on decoupling `Email` and `Member::$unique_identifier_field`? There's still assumptions here which is a pain for people that actually want to use that config option.